packages/ra-core/src/CoreAdminRouter.js

```diff
@@ -135,19 +135,14 @@ export class CoreAdminRouter extends Component {
                 <Switch>
                     {customRoutes
                         .filter(route => route.props.noLayout)
-                        .map((route, index) => (
-                            <Route
-                                key={index}
-                                exact={route.props.exact}
-                                path={route.props.path}
-                                render={props =>
-                                    this.renderCustomRoutesWithoutLayout(
-                                        route,
-                                        props
-                                    )
-                                }
-                            />
-                        ))}
+                        .map((route) => cloneElement(route, {
+                          exact: true,
```

Comment:
why did you override the route's `exact` prop? That's a regression.

Reply:
I though it was written before `<Route exact ... />` but yes after verifying the history it is a mistake again.